apps/mitt-konto/src/MittKonto/Main/Types.purs

```diff
@@ -23,6 +24,7 @@ type State =
   , now :: Date
   , news :: Maybe JSX
   , loginComponent :: Login.Props -> JSX
+  , window :: Maybe (Maybe Window)
```

Comment:
A comment about how `Nothing` and `Just Nothing` differ would be ince.

Reply:
> I think the "Betalningen öppnas i ett nytt fönster" messages should all come with an avbryt button.

I agree. I took a quick shot at this but cancelling the process from the modal is surprisingly convoluted (the same applies at the campaign plugin side). I'd just take the hit from the UX wart instead, as the other option seems to involve rewriting a lot of the handling code, just to get a cancel button.